tests/installer/installer_test.go

```diff
@@ -174,7 +174,7 @@ var _ = Describe("cOS Installer tests", func() {
 				By("Running the cos-installer with a config file")
 				Expect(err).To(BeNil())
 				By("Running the installer")
-				out, err := s.Command("cos-installer --config /tmp/config.yaml /dev/sda")
+				out, err := s.Command("cos-installer --cloud-init /tmp/config.yaml /dev/sda")
```

Comment:
@Itxaka in elemental the installer flag `--config` is translated to `--cloud-init`, was done it on purpose? I did not realize of this mismatch while checking flags as the functionality is there and I can't remember if there was a reason to name it differently. Indeed I prefer `--cloud-init` as we might want to reserve `--config` for installer specific configurations in the future.

Reply:
umm, not on purpose, but probably due to the `config-dir` flag already in elemental and to clarify what type of config is, as on the new elemental we are supposed to have a config file.